types/rails__ujs/utils/event.d.ts

```diff
@@ -0,0 +1,19 @@
+import { SelectorObject } from './dom';
+
+export function fire(obj: EventTarget, name: string, data?: any): boolean;
+
+export function stopEverything(e: Event): void;
+
+export function delegate<K extends keyof HTMLElementEventMap>(
+    element: EventTarget,
+    selector: SelectorObject | string,
+    eventType: K,
+    handler: (this: HTMLElement, ev: HTMLElementEventMap[K]) => any,
```

Comment:
```suggestion
    handler: (ev: HTMLElementEventMap[K]) => any,
```

Isn't it one argument because it is used like `handler.call(target, e)` ?https://github.com/rails/rails/blob/v6.0.3.4/actionview/app/assets/javascripts/rails-ujs/utils/event.coffee#L66